modules/administration-guide/partials/con_gateway.adoc

```diff
@@ -1,7 +1,12 @@
 [id="gateway_{context}"]
 = Gateway
 
-The {prod-short} gateway is a Traefik instance applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.
+The {prod-short} gateway consists of 3 main parts:
+
+* *link:https://github.com/traefik/traefik[Traefik]* instance responsible for requests routing
+* *link:https://github.com/oauth2-proxy/oauth2-proxy[OAuth2-proxy]* ensuring authentication and initiating OAuth flow
+* *link:https://github.com/brancz/kube-rbac-proxy[kube-rbac-proxy]* applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.
```

Comment:
```suggestion
The {prod-short} gateway has 3 main roles:

* Routing requests, using link:https://github.com/traefik/traefik[Traefik].

* Ensuring authentication and initiating OAuth flow, using 
link:https://github.com/oauth2-proxy/oauth2-proxy[OAuth2-proxy].

* Applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource, using link:https://github.com/brancz/kube-rbac-proxy[kube-rbac-proxy].
```

Reply:
Disambiguation needed:

This content implies that the gateway is using the same tools on Kubernetes and OpenShift. 

However, the content in proc_configuring-minikube-github-authentication.adoc seems to imply that  OAuth2-proxy is Minikube or Kubernetes only.

Reply:
on Kubernetes, we use https://github.com/oauth2-proxy/oauth2-proxy and https://github.com/brancz/kube-rbac-proxy. On OpenShift, we use https://github.com/openshift/oauth-proxy/ and https://github.com/openshift/kube-rbac-proxy, which are de-facto same components, only forked and patched to work properly on OpenShift. I'm not sure how to correctly put it here in the docs.